ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
IIUC this is meant to test that `\r` is unchanged. I strongly believe that `fwrite` shouldn't change the bits it's given, so the test should exist in some form. Are you suggesting the form be changed?